griptape/events/event_bus.py

```diff
@@ -32,3 +32,6 @@ def remove_event_listener(self, event_listener: EventListener) -> None:
     def publish_event(self, event: BaseEvent, *, flush: bool = False) -> None:
         for event_listener in self.event_listeners:
             event_listener.publish_event(event, flush=flush)
+
+
+EventBus = _EventBus()
```

Comment:
I could maybe guess at reasons for not doing this, but why not associate fields/methods with the class?

I only ask, because it feels like we're trying to mimic a class with state, but not actually doing it, and I don't exactly understand why

Reply:
chatted about this, resolving